src/core/assert.js

```diff
@@ -91,6 +91,7 @@ function assertion(errorCls, shouldBeTruthy, opt_message, var_args) {
   });
 
   const error = new errorCls(message);
+  error.fromAssert = true;
```

Comment:
Do we need this?

Reply:
I have no idea; trying to make things work and understand why they don't/what depends on what. This PR should probably be in draft for now, sorry.